core/client/hdfs/src/main/java/alluxio/hadoop/AbstractFileSystem.java

```diff
@@ -354,7 +354,7 @@ public FileStatus getFileStatus(Path path) throws IOException {
     return new FileStatus(fileStatus.getLength(), fileStatus.isFolder(),
         getReplica(fileStatus), fileStatus.getBlockSizeBytes(),
         fileStatus.getLastModificationTimeMs(),
-        fileStatus.getCreationTimeMs(), new FsPermission((short) fileStatus.getMode()),
+        fileStatus.getLastAccessTimeMs(), new FsPermission((short) fileStatus.getMode()),
```

Comment:
why is creation time changed to last access time?

Reply:
From HDFS interface this argument is the access time. We just used creation time as a fallback.